addons/sale_product_configurator/static/src/js/product_configurator_widget.js

```diff
@@ -121,15 +137,15 @@ ProductConfiguratorWidget.include({
      * @private
      */
     _openProductConfigurator: function (data, dataPointId) {
+        this.options = undefined;
```

Comment:
why is this needed?

Reply:
It ensures the options are reset when we reOpen the Configurator and that they cannot be applied twice.  I don't think it can happen currently but it is a layer of security :3.